src/Microsoft.PowerShell.ConsoleGuiTools/ShowObjectView.cs

```diff
@@ -0,0 +1,312 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Terminal.Gui;
+using Terminal.Gui.Trees;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+using System.Linq;
+using System.Diagnostics;
+using System.Collections;
+
+namespace OutGridView.Cmdlet
+{
+    internal class ShowObjectView : Window, ITreeBuilder<object>
+    {
+        private readonly TreeView<object> tree;
+        private readonly TreeViewTextFilter<object> filter;
+
+        public bool SupportsCanExpand => true;
+        private StatusItem selectedStatusBarItem;
+        private StatusBar statusBar;
+
+        public ShowObjectView(List<object> rootObjects)
+        {
+            Title = "Show-ObjectTree (Ctrl+Q to quit)";
```

Comment:
I think the look & feel of OCGV and this should be the same. E.g. instead of the "Ctrl-Q to Quit" border title, add a statusbar. Also, OCGV uses Esc to quit as it's more standard.

Reply:
Now uses StatusBar and Escape